polus-infering-splinedist-plugin/plugin.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "SplineDist (Inference) Plugin",
+    "version": "0.0.9",
+    "title": "Inferring SplineDist",
+    "description": "This plugin runs a neural network to identify segments using SplineDist's algorithm",
+    "author": "Madhuri Vihani (Madhuri.Vihani@axleinfo.com),  Hythem Sidky (Hythem.Sidky@nih.gov)",
+    "institution": "National Center for Advancing Translational Sciences, National Institutes of Health",
+    "repository": "https://github.com/labshare/polus-plugins",
+    "website": "https://ncats.nih.gov/preclinical/core/informatics",
+    "citation": "https://www.biorxiv.org/content/10.1101/2020.10.27.357640v1",
+    "containerId": "mmvihani/polus-inferring-splinedist-plugin:0.0.9",
+    "inputs": [
+      {
+        "name": "inpImageDir",
+        "description": "Input Image collection for training",
+        "type": "collection",
+        "options": null,
+        "required": true
+      },
+      {
+        "name": "inpBaseDir",
+        "description": "Input Label collection for training",
+        "type": "genericData",
+        "options": null,
+        "required": true
+      },
+      {
+        "name": "imagePattern",
+        "description": "Pattern of the images in Input",
+        "type": "string",
+        "options": null,
+        "required": false
+      }
+    ],
+    "outputs": [
+      {
+        "name": "outDir",
+        "description": "Output Directory for Plots",
+        "type": "collection",
+        "options": null,
+        "required": true
+      }
+    ],
+    "ui": [
+      {
+        "key": "inputs.inpImageDir",
+        "title": "Input Image Directory: ",
+        "description": "Collection name that contains intensity based images"
+      },
+      {
+        "key": "inputs.inpBaseDir",
+        "title": "Model Directory: ",
+        "description": "Collection name that contains labelled segments"
+      },
```

Comment:
This is just a reminder to edit this however you may edit the inpBaseDir variable from above.